### PR TITLE
Added Angular.JS, Bower, Gulp, and a simple template.

### PR DESCRIPTION
**Pull request by alexkafer**
There are a lot of structural changes here plus a few personal coding conventions. If anything looks weird, please comment and we can discuss. 

This is just installing angular, no grid right now. That shouldn't be too hard and will be my next commit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "type": "git",
     "url": "git+https://github.com/teslaworksumn/proton-sequencer.git"
   },
-  "author": "Ryan Fredlund, Byron Ambright, Joshua Guldberg, Alex Kafer, Kelly Larson, Vladimir Kuksenko",
+  "author": "Ryan Fredlund, Byron Ambright, Joshua Guldberg, Alex Kafer, Kelly Larson, Vladimir Kuksenko, Kayla Engelstad",
```
Comment (alexkafer) [inline package.json:12]: Manually added @kayla-e774 so we don't need to deal with the future merging issue.

**Review by bookdude13 — DISMISSED** [review]
Had to add electron-prebuilt as a dependency and tweak a couple lines to make it run on my machine. Otherwise looks good. I merged your PR into my branch and added my changes. You can take a look, add/merge in, then I will pull to master.

**Comment by alexkafer** [comment]
Looks good to me. I pulled your branch and everything compiles and runs as expected.

**Comment by bookdude13** [comment]
Glad it looks good, but you will also have to merge those changes into this branch yourself, since I don't think I have write access to your fork.

**Comment by bookdude13** [comment]
Resolved with #11

**Pull request closed without merging**